src/ui_nbgl.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "ui.h"
+#include "nbgl_use_case.h"
+
+typedef struct {
```

Comment:
if you don't need to expose `UiContext_t` outside of `ui_nbgl.c` you can probably move the definition of this structure inside `ui_menu_nbg.c` and remove this header file entirely...